### ISSUE TITLE
Competing focus rings in Compose view

### ISSUE THREAD

**Issue by BANOnotIT** [cheeaun/phanpy#1025]
### Bug description

![image](https://github.com/user-attachments/assets/4b3d22cb-6e1e-4ce5-95dc-6a99cdb3a077)
3 elements have focus rings: visibility selector, text area, and language selector.

### To reproduce

Open composition using <kbd>C</kbd> hotkey.

### Expected behavior

Only one element should have focus ring.
